frameworks/hdfs/src/main/java/com/mesosphere/sdk/hdfs/scheduler/Main.java

```diff
@@ -55,28 +57,57 @@ private static SchedulerBuilder createSchedulerBuilder(File yamlSpecFile) throws
         File configDir = yamlSpecFile.getParentFile();
 
         SchedulerConfig schedulerConfig = SchedulerConfig.fromEnv();
+
+        Map<String, String> env = System.getenv();
+
+        String userAuthMapping;
+
+        if (Boolean.valueOf(env.get("TASKCFG_ALL_SECURITY_KERBEROS_ENABLED"))) {
+            String frameworkHostname = EndpointUtils.toAutoIpDomain(rawServiceSpec.getName(), schedulerConfig);
+            HdfsUserAuthMapper authMapper = setupAuthMapper(env, frameworkHostname);
+            authMapper.addUserAuthMappingFromEnv(env, TASKCFG_ALL_AUTH_TO_LOCAL);
+            authMapper.addDefaultUserAuthMapping(JOURNAL_POD_TYPE, "node", JOURNAL_NODE_COUNT);
```

Comment:
From my point of view this interface could be simplified. Would it not make sense to have a single function call (or builder) to build the mapping?